torch/utils/data/datapipes/iter/combining.py

```diff
@@ -6,7 +6,7 @@
 from torch.utils.data.datapipes._decorator import functional_datapipe
 from torch.utils.data.datapipes.datapipe import IterDataPipe
 from torch.utils.data.datapipes.utils.common import check_lambda_fn
-from torch.utils.data._utils.serialization import serialize_fn, deserialize_fn
+from torch.utils.data._utils.serialization import DILL_AVAILABLE, SerializationType, serialize_fn, deserialize_fn
```

Comment:
This is what I was talking about in https://github.com/pytorch/vision/pull/5711#discussion_r839678058. We now need to add `.datapipes.iter.combining` to the list of patched modules.

Reply:
Does https://github.com/pytorch/vision/pull/5711#discussion_r840612921 would temporarily solve the issue?

Reply:
If this is just temporary, I'm fine with patching multiple modules. If we go for https://github.com/pytorch/data/issues/341 then this patching goes away anyway.